CONTRIBUTING.md

```diff
@@ -22,6 +22,62 @@ Run the tests :heavy_check_mark:
 yarn test
 ```
 
+Make sure you include the latest distribution built in your PR commits
+
+```bash
+yarn build && git add .
+```
+
+## Releasing a new version (for core contributors only)
+
+Always release from main
+
+```bash
+git fetch origin && git switch -C main origin/main
+```
+
+Generate the changelog, commit and tag of the release.
+
+```bash
+yarn release
+```
+
+Push the generated commit into `origin/main`
+
+```bash
+git push origin main
+```
+
+Push the generated version tag into `origin/main`.
+
+```bash
+git push --follow-tags origin main
+```
+
+This Github action follows the [recommended release workflow](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md#recommendations)
+
+### Releasing minor and bath versions
```

Comment:
bath versions looks a typo

Reply:
Please approve when you feel that it is ready